homework-13/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Homework 13</title>
+</head>
+
+<body>
+    <div class="container">
+        <section class="section-1">
+            <input type=" text" class="input1" id="input"><button class="btn" id="btn">Start Timeout</button>
+            <p class="paragraph"></p>
+        </section>
+        <section class="section-2">
+            <input type="date" class="input2" id="input2">
+            <button class="btn-campining" id="btn-campaign">Start Campaign</button>
+            <p id="endtime"></p>
+        </section>
+        <section class="section-3">
+            <select name="model" id="model">
+                <label for="phone-select">Model of phone:</label>

```

Comment:
Labels should be outside the select here and below
There is no input with `phone-select` id